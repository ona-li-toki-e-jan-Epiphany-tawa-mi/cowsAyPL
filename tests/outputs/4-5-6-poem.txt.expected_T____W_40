/¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯\
| I want to fall. Infi |
| nitely far. Onto a b |
| ed of stars.         |
\______________________/
                    \                  
                     \                 
                       ^__^            
                       (oo)\_______    
                       (__)\       )\/\
                        __ ||----w |   
                           ||     ||   
