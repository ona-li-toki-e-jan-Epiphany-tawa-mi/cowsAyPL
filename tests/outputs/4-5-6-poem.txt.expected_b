/¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯\
| + b I want to fall. Infinitely far. Onto |
|  a bed of stars.                         |
\__________________________________________/
                                        \                  
                                         \                 
                                           ^__^            
                                           (oo)\_______    
                                           (__)\       )\/\
                                               ||----w |   
                                               ||     ||   
