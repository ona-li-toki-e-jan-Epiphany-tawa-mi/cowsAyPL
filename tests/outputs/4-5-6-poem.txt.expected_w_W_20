/¯¯¯¯¯¯¯¯¯¯¯¯\
| I want to  |
| fall. Infi |
| nitely far |
| . Onto a b |
| ed of star |
| s.         |
\____________/
          \                  
           \                 
             ^__^            
             (OO)\_______    
             (__)\       )\/\
                 ||----w |   
                 ||     ||   
