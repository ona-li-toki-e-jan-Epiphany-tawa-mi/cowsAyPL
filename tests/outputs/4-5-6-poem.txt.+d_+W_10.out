/¯¯¯¯¯¯¯¯¯¯¯¯\
| I want to  |
| fall. Infi |
| nitely far |
| . Onto a b |
| ed of star |
| s.         |
\____________/
        \
         \
           ^__^
           (XX)\_______
           (__)\       )\/\
            U  ||----w |
               ||     ||

